app/controllers/api/v1/users_controller.rb

```diff
@@ -13,7 +13,8 @@ def create
         user = User.new(user_params)
 
         if user.save
-          user.welcome_user
+          user.invite_to_slack
+          user.add_to_send_grid
```

Comment:
these should probably occur after `sign_in` right? Unsure why `user.welcome_user` was listed before `UserMailer.welcome(user).deliver unless user.invalid?`